samples/bluetooth/broadcast_audio_sink/Kconfig

```diff
@@ -28,4 +28,11 @@ config SCAN_DELAY
 	  Time to advertise connectable for a Broadcast Assistant to connect
 	  before starting scanning for Broadcast Sources.
 
+config SD_RECEIVE_STATE_OVERWRITE_OLDEST
```

Comment:
This should be defined for the stack (as that is where it is used), rather for the sample. Add it somewhere here: https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/audio/Kconfig.bap#L213-L238

And be renamed to `CONFIG_BT_BAP_SCAN_DELEGATOR_OVERWRITE_OLDEST`